digdag-standards/src/main/java/io/digdag/standards/operator/WaitOperatorFactory.java

```diff
@@ -0,0 +1,155 @@
+package io.digdag.standards.operator;
+
+import com.google.common.base.Optional;
+import com.google.inject.Inject;
+import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigElement;
+import io.digdag.client.config.ConfigException;
+import io.digdag.spi.Operator;
+import io.digdag.spi.OperatorContext;
+import io.digdag.spi.OperatorFactory;
+import io.digdag.spi.TaskRequest;
+import io.digdag.spi.TaskResult;
+import io.digdag.spi.TaskExecutionException;
+
+import io.digdag.util.Durations;
+import io.digdag.util.Workspace;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.concurrent.TimeUnit;
+
+
+public class WaitOperatorFactory
+        implements OperatorFactory {
+
+    @Inject
+    public WaitOperatorFactory()
+    {
+    }
+
+    @Override
+    public String getType()
+    {
+        return "wait";
+    }
+
+    @Override
+    public Operator newOperator(OperatorContext context)
+    {
+        return new WaitOperator(context);
+    }
+
+    private static class WaitOperator
+            implements Operator
+    {
+        private static final String WAIT_START_TIME_PARAM = "wait_start_time_millis";
+
+        private static final Logger logger = LoggerFactory.getLogger(WaitOperator.class);
+
+        private final OperatorContext context;
+        private final TaskRequest request;
+        private final Workspace workspace;
+
+
+        private WaitOperator(OperatorContext context)
+        {
+            this.context = context;
+            this.request = context.getTaskRequest();
+            this.workspace = Workspace.ofTaskRequest(context.getProjectPath(), request);
+        }
+
+        private Duration duration(Config config)
+        {
+            Duration duration;
+            try {
+                duration = Durations.parseDuration(config.get("_command", String.class));
+            }
+            catch (RuntimeException re) {
+                throw new ConfigException("Invalid configuration", re);
+            }
+            logger.debug("wait duration: {}", duration);
+            return duration;
+        }
+
+        private boolean blocking(Config config)
+        {
+            boolean blocking = config.get("blocking", boolean.class, false);
+            logger.debug("wait blocking mode: {}", blocking);
+            return blocking;
+        }
+
+        private Optional<Duration> pollInterval(Config config)
+        {
+            Duration pollInterval;
+            try {
+                Optional<String> pollIntervalStr = config.getOptional("poll_interval", String.class);
+                if (!pollIntervalStr.isPresent()) {
+                    return Optional.absent();
+                }
+                pollInterval = Durations.parseDuration(pollIntervalStr.get());
```

Comment:
I guess we may make the codes simple by using Optional.transform().
But not strong opinion.

```
       try {
            Optional<Duration> pollInterval = config
                    .getOptional("poll_interval", String.class)
                    .transform(Durations::parseDuration);
            if (pollInterval.isPresent()) {
                logger.debug("wait poll_interval: {}", pollInterval.get());
            }
            return pollInterval;
        }
        catch (RuntimeException re) {
            throw new ConfigException("Invalid configuration", re);
        }
```